contracts/cw721-base/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name          = "cw721-base"
 description   = "Basic implementation cw721 NFTs"
 authors       = [
+  "mr-t <mr-t@arkprotocol.io>",
   "Ethan Frey <ethanfrey@users.noreply.github.com>",
   "Orkun Külçe <orkun@deuslabs.fi>",
 ]
```

Comment:
hope this is ok, adding me as main contributor :)

Reply:
You literally contributed lol

Reply:
:heart: